### PR TITLE
adding multibanco apm

### PR DESCRIPTION
**Pull request by aditya27marri**
### Description

Adding multiblanco apm to spb

### Dependent Changes (if applicable)

❤️  Thank you!

### CODE REVIEW

**Comment by gregjopa** [comment]
Hi @aditya27marri, can we work on fixing this typo for `multibanco` here before merging this PR? https://github.com/paypal/paypal-sdk-constants/pull/81 cc @arielxin

**Comment by aditya27marri** [comment]
> Hi @aditya27marri, can we work on fixing this typo for `multibanco` here before merging this PR? [paypal/paypal-sdk-constants#81](https://github.com/paypal/paypal-sdk-constants/pull/81) cc @arielxin

SUre will do it Greg

**Comment by aditya27marri** [comment]
https://github.com/paypal/paypal-sdk-constants/pull/85 I changed it the name as required @gregjopa

**Review by wsbrunson — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "@paypal/common-components": "^1.0.16",
     "@paypal/installments": "^1.0.0",
     "@paypal/sdk-client": "^4.0.156",
-    "@paypal/sdk-constants": "^1.0.107",
+    "@paypal/sdk-constants": "^1.0.117",
     "@paypal/sdk-logos": "^1.0.26",
```
[the remark this answers is not in the document]
Comment (aditya27marri) [reply package.json]: I updated it. Please check now

**Review by wsbrunson — APPROVED** [review]

**Review by gregjopa — APPROVED** [review]

**Pull request merged**